src/component-systems/ClippingSystem.js

```diff
@@ -48,9 +71,11 @@ export class ClippingSystem extends System {
         });
     }
 
+// TODO - is this function still needed? i dont see it called at all?
```

Comment:
@michaelthatsit - quick question for you

Reply:
Yes still needed! this is called on an event whenever a new entity is added. all systems have it but not all of them utilize it.